testutil/ciao-down/cloudinit.go

```diff
@@ -273,7 +273,14 @@ write_files:
      printf "\n"
      printf "\n"
      printf "Your go code is at {{.GoPath}}\n"
-     printf "You can also edit your code on your host system"
+     printf "You can also edit your code on your host system \n"
+     printf "To build cc-oci-runtime from sources \n"
+     printf "git clone https://github.com/01org/cc-oci-runtime \n"
+     printf "cd cc-oci-runtime \n"
+     printf "./autogen.sh --with-cc-kernel=/usr/share/clear-containers/vmlinux.container --with-cc-image=/usr/share/clear-containers/clear-containers.img \n"
+     printf "make \n"
+     printf "make check \n"
+     printf "make install \n"
```

Comment:
@markdryan you should not need to restart docker or any other service. I left out make install. 
Let me add that.